src/components/Redux/RocketsSlice/RocketSlice.js

```diff
@@ -1,13 +1,37 @@
-import { createSlice } from '@reduxjs/toolkit';
-import data from '../../TempRocketData';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+/* import data from '../../TempRocketData'; */
+
+const url = 'https://api.spacexdata.com/v3/rockets';
 
 const initialState = {
-  RocketDatile: data,
+  RocketDatile: [],
+  loading: false,
+  error: null,
 };
 
+export const rocketItems = createAsyncThunk('rocket', async () => {
+  const response = await fetch(url);
+  const result = response.json();
+  return result;
```

Comment:
Great work on fetching data from the API using AsyncThunk. Remember to restructure the obtained array to only retrieve the: 
id
name
type
flickr_images